RNReanimated.podspec

```diff
@@ -14,6 +14,7 @@ folly_compiler_flags = folly_flags + ' ' + '-Wno-comma -Wno-shorten-64-to-32'
 boost_compiler_flags = '-Wno-documentation'
 fabric_flags = fabric_enabled ? '-DRCT_NEW_ARCH_ENABLED' : ''
 version_flag = '-DREANIMATED_VERSION=' + reanimated_package_json["version"]
+use_hermes = ENV['USE_HERMES'] == '1'
```

Comment:
Another option is `hermes_enabled` which would be consistent with already existing `fabric_enabled` flag.